File: C# Part 2/03.Methods/06.FirstLargerThanNeighbours.cs
using System;
using System.Linq;

namespace FirstLargerThanNeighbours
{
    class FirstLargerThanNeighbours
    {
        static void Main()
        {
            int sizeOfArray = Convert.ToInt32(Console.ReadLine());
            int[] input = Console.ReadLine().Split(' ').Select(x => Convert.ToInt32(x)).ToArray();
            Console.WriteLine(Check(input));
        }

        static int Check(int[] input)
        {
            int count = 0;
            for (int i = 1; i < input.Length - 1; i++) if (input[i] > input[i - 1] && input[i] > input[i + 1])
            {
                return i;
            }
            return -1;
        }
    }
}

File: JavaScript Fundamentals/08. Functions/06_FirstLargerThanNeighbours.js
function solve(args) {
    args = args[0].split('\n');
    var n = +args[0];
    args = args[1].split(' ');


    var count = 0;
    for (var i = 1; i < n - 1; i += 1) {
        var current = +args[i];
        if ((current > +args[i + 1]) && (current > +args[i - 1])) {
            return i;
        }
    }
    return -1;

}

solve(['6\n-26 -25 -28 31 2 27']);
